modules/afstomp/afstomp.c

```diff
@@ -296,8 +295,14 @@ afstomp_worker_publish(STOMPDestDriver *self, LogMessage *msg)
   stomp_frame_add_header(&frame, "destination", self->destination);
   if (self->ack_needed)
     {
-      g_snprintf(seq_num, sizeof(seq_num), "%i", self->super.worker.instance.seq_num);
-      stomp_frame_add_header(&frame, "receipt", seq_num);
+      /*
+       * We check the server's response before sending a new frame to it.
+       * Because of this, we do not need a unique receipt header.
+       *
+       * This changes if multiple workers and/or batching support is introduced.
+       * Make sure to use a unique receipt-id if one of the above gets implemented.
+       */
+      stomp_frame_add_header(&frame, "receipt", "0");
     };
 
   LogTemplateEvalOptions options = {&self->template_options, LTZ_SEND, self->super.worker.instance.seq_num, NULL, LM_VT_STRING};
```

Comment:
Should we set `seq_num` in the `LogTemplateEvalOptions` to 0 too so `$SEQNUM` macro will be 0 consistently?

Reply:
I would not do that, because maybe someone would need the actual `$SEQNUM` in the body of the message.

IMO the fact that previously `seq_num` was a unique value, and that we needed a unique value for each message in the `receipt` header, was only a coincidence. Both `seq_num` and `receipt` header have their own uses, `seq_num` is a message metadata, `receipt` is used for acknowledgement in the STOMP protocol.

With this commit I only wanted to remove the coincidental connection between the two.